apps/rush-lib/assets/rush-init/rush.json

```diff
@@ -64,20 +64,6 @@
      */
     /*[LINE "DEMO"]*/ "strictPeerDependencies": true,
 
-    /**
-     * Configures the strategy used to select versions during installation.
-     *
-     * This feature requires PNPM version 3.1 or newer.  It corresponds to the "--resolution-strategy" command-line
-     * option for PNPM.  Possible values are "fast" and "fewer-dependencies".  PNPM's default is "fast", but this may
-     * be incompatible with certain packages, for example the "@types" packages from DefinitelyTyped.  Rush's default
-     * is "fewer-dependencies", which causes PNPM to avoid installing a newer version if an already installed version
-     * can be reused; this is more similar to NPM's algorithm.
-     *
-     * After modifying this field, it's recommended to run "rush update --full" so that the package manager
-     * will recalculate all version selections.
-     */
-    /*[LINE "HYPOTHETICAL"]*/ "resolutionStrategy": "fast",
```

Comment:
@D4N14L explained that this setting was deprecated by PNPM and has no effect with PNPM >5.0.

Good ways to handle this:
1. Remove the setting from Rush, so that it is now a hard error if anyone tries to use it.  This would be a "breaking change" in a minor version of Rush, but Rush versions do not strictly follow SemVer, so maybe that's fine
2. Plan to remove this setting in Rush 6.x.  For Rush 5.x, report a hard error only if the `pnpmVersion` does not support this feature.  And update the **rush.json** comments to reflect reality.

Not good ways to handle this:
3. Remove the documentation from **rush.json**, so that nobody knows what this setting does, but silently accept the setting and implement its full behavior for older PNPM